mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerNoScan.java

```diff
@@ -210,28 +210,25 @@ public void testCreateDropTable() throws TException, IOException, InterruptedExc
       shell.executeStatement("DROP TABLE customers");
 
       // Check if the table was really dropped even from the Catalog
-      AssertHelpers.assertThrows(
-          "should throw exception",
-          NoSuchTableException.class,
-          "Table does not exist",
-          () -> {
-            testTables.loadTable(identifier);
-          });
+      Assertions.assertThatThrownBy(
+              () -> {
```

Comment:
nit: the { } aren't necessary here and further below

Reply:
Done!